packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -0,0 +1,407 @@
+};
+const test = anyTest as TestFn<Awaited<ReturnType<typeof makeZoeTestContext>>>;
+
+test.before(async t => {
+  t.context = await makeZoeTestContext(t);
+});
+
+test.serial('normal running of committee', async t => {
+  const { advanceTimeBy, storage, getUpdatedDebtLimit, governanceDriver } =
+    t.context;
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { VaultFactory, economicCommittee } = agoricNamesRemotes.instance;
+  const { ATOM: collateralBrand, IST: debtBrand } = agoricNamesRemotes.brand;
+
+  const committee = governanceDriver.ecMembers;
+
+  t.log('Accepting all invitations for original committee');
+  await null;
+  for (const member of committee) {
+    await member.acceptCharterInvitation(offerIds.propose.outgoing);
+    await member.acceptCommitteeInvitation(offerIds.vote.outgoing);
+  }
+
+  t.log('Proposing a question using first wallet');
+  await governanceDriver.proposeParams(
+    VaultFactory,
+    { DebtLimit: { brand: debtBrand, value: 100_000_000n } },
+    { paramPath: { key: { collateralBrand } } },
+    committee[0],
+    getQuestionId(1),
+    offerIds.propose.outgoing,
+  );
+
+  t.log('Checking if question proposal passed');
+  t.like(committee[0].getLatestUpdateRecord(), {
+    status: { id: getQuestionId(1), numWantsSatisfied: 1 },
+  });
+
+  t.log('Voting on question using first 3 wallets');
+  await governanceDriver.enactLatestProposal(
+    committee,
+    getVoteId(1),
+    offerIds.vote.outgoing,
+  );
+
+  t.log('Checking if votes passed');
+  for (const w of committee.slice(0, 3)) {
+    t.like(w.getLatestUpdateRecord(), {
+      status: { id: getVoteId(1), numWantsSatisfied: 1 },
+    });
+  }
+
+  t.log('Waiting for period to end');
+  await advanceTimeBy(1, 'minutes');
+
+  t.log('Verifying outcome');
+
+  const lastOutcome = await governanceDriver.getLatestOutcome();
+  console.log(lastOutcome);
+  t.deepEqual(getUpdatedDebtLimit(), 100_000_000n);
+  t.assert(lastOutcome.outcome === 'win');
+});
+
+test.serial(
+  'check high priority senders before replacing committee',
+  async t => {
+    const { storage } = t.context;
+
+    const data: any = storage.toStorage({
+      method: 'children',
+      args: [highPrioritySenderKey],
+    });
+    t.deepEqual(data.sort(), [...wallets].sort());
+  },
+);
+
+test.serial('replace committee', async t => {
+  const { buildProposal, evalProposal, storage } = t.context;
+
+  const preEvalAgoricNames = makeAgoricNamesRemotesFromFakeStorage(storage);
+  await evalProposal(
+    buildProposal(
+      '@agoric/builders/scripts/inter-protocol/replace-electorate-core.js',
+      ['BOOTSTRAP_TEST'],
+    ),
+  );
+  await eventLoopIteration();
+
+  const postEvalAgoricNames = makeAgoricNamesRemotesFromFakeStorage(storage);
+
+  t.not(
+    preEvalAgoricNames.instance.economicCommittee,
+    postEvalAgoricNames.instance.economicCommittee,
+  );
+});
+
+test.serial(
+  'check high priority senders after replacing committee',
```

Comment:
Since there's some question of whether the price feeds have been notified of the new committee, how about we test governing them?

 1. using API governance, add an oracle operator
 2. check that the new oracle operator address is in the list of high priority senders

I'm not sure there's any other way to observe that an EC decision that affects a price feed has taken effect.

Reply:
tests for each individual contract are in the next PR. I've put them there since it would be better to perform that test when we have created new instances of both the charter and the committee